data/json/items/containers/containers.json

```diff
@@ -1271,6 +1271,33 @@
     "symbol": ")",
     "color": "brown"
   },
+  {
+    "id": "box_oversize_reinforced",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "reinforced oversize cardboard box", "str_pl": "reinforced oversize cardboard boxes" },
+    "looks_like": "box_small",
+    "description": "An oversize cardboard box, doubled up with a second layer of cardboard and lots of duct tape.  It can't be folded anymore.",
+    "ascii_picture": "box_oversize",
+    "weight": "4 kg",
```

Comment:
I think the weight should be a little more than two boxes stuck together if the recipe uses 8 yards of duct tape. With our current weight of duct tape at $\frac{3500 mg}{20 yards} = 17.5 mg/yard$, there should be an extra 1400 mg of weight.

Reply:
I have changed the weight from 4kg to 4002g which tbh is, idk if that was necessary but I appreciate that you took the time to look at what I made and provide feedback.